xbmc/interfaces/python/XBPython.cpp

```diff
@@ -581,6 +581,10 @@ bool XBPython::OnScriptInitialized(ILanguageInvoker *invoker)
       CLog::Log(LOGDEBUG, "PYTHONHOME -> %s", CSpecialProtocol::TranslatePath("special://frameworks").c_str());
       CLog::Log(LOGDEBUG, "PYTHONPATH -> %s", CSpecialProtocol::TranslatePath("special://frameworks").c_str());
     }
+
+#if defined(TARGET_DARWIN)
```

Comment:
mhh on osx there is not even a place i know of to set environment variables for normal gui app bundles. The only thing i can think off is to check the env before setting it to have it as fallback. Though i am 99% sure that there is no easy way for the user to set an environment for kodi from extern. On iOS its even more unlikely that this var overwrites anything explicitly set by the user. Would a check before setting it in code acceptable?